tests/test_types.py

```diff
@@ -12,6 +12,8 @@ def test_abstract_ints():
     assert not issubclass(t.uint8_t, t.int_t)
     assert t.int_t._signed is True
     assert t.uint_t._signed is False
+    assert t.int_t._bigendian is False
+    assert t.be_int_t._bigendian is True
```

Comment:
What do you think about `t.int_t_be`? Having a `be_` prefix makes it sound a little strange when read aloud.

Reply:
Ha-ha, never thought of it!
I have no particular preference, and I couldn't find any existing convention. Will change.